library/proc_macro/Cargo.toml

```diff
@@ -5,3 +5,7 @@ edition = "2021"
 
 [dependencies]
 std = { path = "../std" }
+# Workaround: when documenting this crate rustdoc will try to load crate named
+# `core` when resolving doc links. Without this line a different `core` will be
+# loaded from sysroot causing duplicate lang items and other similar errors.
+core = { path = "../core" }
```

Comment:
I wish I'd thought of this a year ago :( this is much simpler than https://github.com/rust-lang/rust/pull/86876 or https://github.com/rust-lang/rust/pull/75176. Good idea!

Reply:
Both https://github.com/rust-lang/rust/pull/86876 and https://github.com/rust-lang/rust/pull/75176 were helpful for other reasons though, like closing #56935 and improving performance.

Reply:
Ooh, this might help with removing `BadImplStripper`...